### PR TITLE
Prepare openQA to test GNOME Next Live image flavors

### PR DESCRIPTION
**Pull request by DimStar77**

### CODE REVIEW

File: lib/main_common.pm [hunk 1]
```diff
@@ -225,7 +225,7 @@ sub kdestep_is_applicable {
 }
 
 sub consolestep_is_applicable {
-    return !get_var("INSTALLONLY") && !get_var("DUALBOOT") && !get_var("RESCUECD");
+    return !get_var("INSTALLONLY") && !get_var("DUALBOOT") && !get_var("RESCUECD") && !check_var('FLAVOR', 'Gnome-Live');
```
Comment (okurz) [inline lib/main_common.pm]: why should that not apply for gnome-live when we have that for the kde live media?

File: tests/x11/evolution.pm [hunk 2]
```diff
@@ -17,10 +17,12 @@ use testapi;
 
 sub run() {
     x11_start_program("evolution");
+    if ((get_var('FLAVOR') =~ /Gnome-Live/) && check_screen("evolution-preview-release", 20)) {
+        send_key "alt-o";   # OK
+    }
     if (check_screen "evolution-default-client-ask", 20) {
         assert_and_click "evolution-default-client-agree";
     }
-    assert_screen 'test-evolution-1', 30;
```
Comment (okurz) [inline tests/x11/evolution.pm]: why?

File: tests/x11/evolution.pm [hunk 3]
```diff
@@ -17,10 +17,12 @@ use testapi;
 
 sub run() {
     x11_start_program("evolution");
+    if ((get_var('FLAVOR') =~ /Gnome-Live/) && check_screen("evolution-preview-release", 20)) {
+        send_key "alt-o";   # OK
+    }
```
Comment (okurz) [inline tests/x11/evolution.pm]: better not waste 20 seconds waiting

```perl
my @tags = qw(test-evolution-1 evolution-default-client-ask);
push(@tags, 'evolution-preview-release') if get_var('FLAVOR') =~ /Gnome-Live/;
do {
    assert_screen(\@tags);
    send_key 'alt-o' if (match_has_tag('evolution-preview-release'));
    assert_and_click 'evolution-default-client-agree' if (match_has_tag('evolution-default-client-ask'));
} until (match_has_tag('test-evolution-1'));
```

File: tests/x11/evolution.pm [hunk 4]
```diff
@@ -17,11 +17,13 @@ use testapi;
 
 sub run() {
     x11_start_program("evolution");
+    if ((get_var('FLAVOR') =~ /Gnome-Live/) && check_screen("evolution-preview-release", 20)) {
+        send_key "alt-o";    # OK
+    }
```
Comment (okurz) [inline tests/x11/evolution.pm]: better use an approach as in `tests/x11/kontact.pm`

```perl
my @tags = qw(test-evolution-1 evolution-default-client-ask);
push(@tags, 'evolution-preview-release') if get_var('FLAVOR') =~ /Gnome-Live/;
do {
    send_key 'alt-o' if match_has_tag('evolution-preview-release');
    assert_and_click 'evolution-default-client-agree' if match_has_tag('evolution-default-client-ask');
} until (match_has_tag('test-evolution-1'));
```

File: lib/main_common.pm (hunk 1 above)
Comment (DimStar77) [inline lib/main_common.pm]: It's a bunch of waste-of time to execute all those console tests for the GNOME Next media; nobody is going to be interested in touching those failures; so why test them?

File: tests/x11/evolution.pm (hunk 2 above)
Comment (DimStar77) [inline tests/x11/evolution.pm]: hmm. that seems weird and unintational.. this is the main evo screen to be found; must be a mistake

File: lib/main_common.pm (hunk 1 above)
Comment (okurz) [inline lib/main_common.pm]: yeah, sure. Then maybe just ignore them for the kde-media as well? I don't see why we should make a difference for kde/gnome live media

File: tests/x11/evolution.pm (hunk 4 above)
Comment (DimStar77) [inline tests/x11/evolution.pm]: I like that... thanks... will update evolution.pm - might even consider allowing the 'preview release' in Stagings; as we see it there when we submit Release Candidates (never allowed to checkin to TW though)

File: lib/main_common.pm (hunk 1 above)
Comment (DimStar77) [inline lib/main_common.pm]: sure - but that should be another commit than "Prepare openQA to the GNOM Next Live image"

Comment (okurz) [inline lib/main_common.pm]: one PR can have more than one commit ;-)

Comment (DimStar77) [inline lib/main_common.pm]: sure sure... will add one for you;

**Comment by DimStar77** [comment]
addressed the review points of @okurz and restructured the code a bit more, to make better use of is_gnome_next; like this we only match the FLAVOR in one single place
